src/Exception/FailedToProduceMessages.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KafkaRest\Exception;
+
+use RuntimeException;
+
+final class FailedToProduceMessages extends RuntimeException
+{
+    /** @var mixed[] */
+    public $retryable;
+
+    /** @var mixed[] */
```

Comment:
```suggestion
    /** @var mixed[]|null */
```

Reply:
it's never null unless you use default constructor, which is a misuse (you wouldn't instantiate the class yourself anyway)